build/package/Microsoft.DotNet.Cli.Installer.PKG.targets

```diff
@@ -1,10 +1,132 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-    
-    <UsingTask TaskName="PkgTargets" AssemblyFile="$(CLIBuildDll)" />
+    <UsingTask TaskName="ReplaceFileContents" AssemblyFile="$(CLIBuildDll)" />
+  
+    <Target Name="SetupPkgInputsOutputs">
+      <PropertyGroup>
+        <InstallerOutputDirectory>$(PackagesDirectory)</InstallerOutputDirectory>
+        <PkgIntermediateDirectory>$(IntermediateDirectory)/pkgs/$(SdkVersion)</PkgIntermediateDirectory>
 
-    <Target Name="GeneratePkgs" >
-        <PkgTargets />
+        <SdkInstallerFile>$(InstallerOutputDirectory)/$(ArtifactNameWithVersionSdk)$(InstallerExtension)</SdkInstallerFile>
+        <CombinedFrameworkSdkHostInstallerFile>$(InstallerOutputDirectory)/$(ArtifactNameWithVersionCombinedHostHostFxrFrameworkSdk)$(BundleExtension)</CombinedFrameworkSdkHostInstallerFile>
+
+        <!-- Rename is intentional to reduce confusion over generated vs downloaded pkg's -->
```

Comment:
Yeah I was thinking that as well since I've now done this in two places, seems reasonable?